### PR TITLE
[AT-69] Update Prized Tournaments List in Tournament HP

### PR DESCRIPTION
**Pull request by Idan-Garay**
Steps:
1. Visit /boardgame-tournaments page
2. Under "Prized Tournaments for Every Boardgame" section, there should only be Prized tournaments and filler tournaments i.e. Wingspan.

### CODE REVIEW

File: app/boardgame-tournaments/page.tsx [hunk 1]
```diff
@@ -86,6 +85,24 @@ const TournamentsPage = async () => {
...
+        title: item.title,
+        date: item.date,
+        uri: item.uri,
+        featuredImageUrl: item.featuredImage.node.sourceUrl ?? null,
+        totalPrize: item.template.tournamentDetail.totalPrize,
+        entryPrice: item.template.tournamentDetail.entryPrice,
+        minPlayerCount: item.template.tournamentDetail.minPlayerCount,
+        maxPlayerCount: item.template.tournamentDetail.maxPlayerCount,
+        status: item.template.tournamentDetail.status,
+      };
```
Comment (idiglove) [inline app/boardgame-tournaments/page.tsx]: better to do optional chaining in case something happens in the backend, really great effort!

Comment (idiglove) [reply app/boardgame-tournaments/page.tsx]: You also can destructure the object

const {totalPrize, entryPrice.. etc} = item?.template?.tournamentDetail ?? {}

**Review by idiglove — COMMENTED** [review]
nice just some comments! i have to test it too

File: app/boardgame-tournaments/page.tsx (hunk 1 above)
Comment (Idan-Garay) [reply app/boardgame-tournaments/page.tsx]: I appreciate the feedback, this looks a lot cleaner

File: app/_components/Content/ImgGrid/index.tsx [hunk 2]
```diff
@@ -8,6 +8,18 @@ import { GridItem, GridItemProps } from "./components/GridItem";
...
+    items.length < 16
+      ? items.concat(
+          Array(16 - items.length).fill({
+            title: "Wingspan",
+            featuredImageUrl: "/img/wingspan.jpg",
+            status: "Coming Soon",
+          })
+        )
+      : items;
+  console.log(normalizedItems);
```
Comment (idiglove) [inline app/_components/Content/ImgGrid/index.tsx]: remove console log pls thanks

**Review by idiglove — APPROVED** [review]
have some comments here and discord, approving, and just merge when you can :)

**Pull request merged**